loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -594,6 +596,70 @@ module BABYLON.GLTF2 {
             var material = getMaterial(runtime, primitive.material);
             multiMat.subMaterials.push(material);
 
+            // Morph Targets
+            if (primitive.targets !== undefined) {
+                morphTargetManager = new BABYLON.MorphTargetManager();
+
+                for (var index2 = 0; index2 < primitive.targets.length; index2++) {

```

Comment:
Better, like smaller (j, k, l), so it's faster to load or better, like bigger (targetIndex, bufferIndex, etc) for clarity?